slick.grid.js

```diff
@@ -1533,6 +1535,12 @@ if (typeof Slick === "undefined") {
       columnsById = {};
       for (var i = 0; i < cols.length; i++) {
         var m = cols[i];
+
+        if (m.json && m.json.isHidden) {
+          m.width = 0;

```

Comment:
Yup to last question.

As you've probably already seen, there is short circuiting code elsewhere to prevent drawing the column headers & rows.